src/main/webapp/index.html

```diff
@@ -10,7 +10,8 @@
     </head>
     <body onload="initBody();">
         <h2>Hello World!</h2>
-        <button class="button" onclick="updateJobDatabase()">Fetch Info</button>
+        <button class="button" onclick="updateProjectDatabase()">Fetch Info</button>
+        <button class="button" onclick="getJobsFromProject('bt-dataflow-sql-demo')">Show info</button>
```

Comment:
Hardcoded project id. I think this should be read from `config.js`

Reply:
Changed this in https://github.com/googleinterns/step229-2020/pull/22/commits/3ba0a2ff46e7491182f0eb136bf26bd95bc3c4db
Thanks :D